extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -268,16 +267,18 @@ function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
         DicomMetadataStore.studyLoaded(study);
       }
 
-      // Google Cloud Healthcare doesn't return StudyInstanceUID, so we need to add
-      // it manually here
-      seriesSummaryMetadata.forEach(aSeries => { aSeries.StudyInstanceUID = StudyInstanceUID })
-
       DicomMetadataStore.addSeriesMetadata(seriesSummaryMetadata, madeInClient);
 
-      const numberOfSeries = seriesPromises.length;
-      seriesPromises.forEach(async (seriesPromise, index) => {
-        const instances = await seriesPromise;
-        storeInstances(instances);
+      const numberOfSeries = seriesSummaryMetadata.length;
+      seriesSummaryMetadata.forEach(async (series, index) => {
+        const instancesResponse = await nlApi.get("http://localhost:8000/api/instances/?series_id=" + series.id);
```

Comment:
the hostname needs to be dynamically discovered. localhost:8000 will only work in a local dev environment

Reply:
is there another example of dynamically discovering the host which Adin can reference? cc @bigshoesdev